addons/mail/models/mail_alias_mixin_optional.py

```diff
@@ -70,6 +70,12 @@ def create(self, vals_list):
                 alias_vals.update(self.env[self._name].with_context(
                     default_alias_domain_id=company.alias_domain_id.id,
                 )._alias_get_creation_values())
+                existing_alias = self.env['mail.alias'].search([
```

Comment:
It is not the purpose of the alias mixin to do this. Issue comes with accounting that tries to automatically create aliases; we don't have that issue in other addons (yet). So this code is heavily complicated for no use case here.

Reply:
The idea behind doing this here was to generally avoid errors happening when optional aliases are automatically created, since it's not the client's fault, but if it's specific to accounting I will change the approach, thank you